commands/create-account.js

```diff
@@ -32,6 +38,26 @@ module.exports = {
 
 async function createAccount(options) {
```

Comment:
You should add a check that the suffix matches the network that the user is talking to.

Reply:
other than that, looks good.